File: bulk/java/Xxe01.java
// CWE-611: XXE via DocumentBuilder
import javax.xml.parsers.*;
import org.w3c.dom.*;
import java.io.*;
public class Xxe01 {
    public Document parse(InputStream is) throws Exception {
        DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
        return builder.parse(is); // XXE enabled by default
    }
}

File: bulk/csharp/Xxe01.cs
// CWE-611: XXE via XmlDocument without disabling DTD
using System.Xml;
public class Xxe01 {
    public XmlDocument Parse(string xml) {
        var doc = new XmlDocument();
        doc.LoadXml(xml); // DTD processing enabled
        return doc;
    }
}
